ci/pipeline

```diff
@@ -183,7 +198,7 @@ def master(): Unit = {
   val version = build()
 
   // Uploads
-  val maybeArtifact = uploadTarballPackagesToS3(version)
+  val maybeArtifact = uploadTarballPackagesToS3(version, "snapshots")
```

Comment:
Should we change it to `builds` then?

Reply:
yeah... this was a PR for releases...   I don't know the complete consequence of changing the name of snapshots and thought it would be follow up work (like native packages).   This is likely really really simple and I'll change it.

Reply:
Ok, we can follow up if we want to.